pkg/vendir/directory/directory.go

```diff
@@ -4,7 +4,10 @@
 package directory
 
 import (
+	"crypto/sha256"
+	"encoding/hex"
 	"fmt"
+	"gopkg.in/yaml.v3"
```

Comment:
Can we use `sigs.k8s.io/yaml` instead? Just to keep it consistent

Reply:
Sure